setup.cfg

```diff
@@ -38,7 +38,7 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    docutils>=0.15
+    docutils>=0.15,<0.17
```

Comment:
why less than 0.17? this is the latest version and supported by myst-parser

Reply:
must of misread `myst-parser::setup.cfg` -- I'll update this.

Reply:
(in fact myst-parser specifically tests against 0.17)

Reply:
you may well get changes in the HTML output doing this FYI (if we are testing against those); there were changes from div+class -> semantic HTML tags (hence why myst-parser tests pin docutils to 0.17)

Reply:
👍 I think most of the tests for `myst-nb` are at the `xml` and `ipynb` level. Running the `tox` environment for `py38-sphinx4` installs `docutils=0.17` so not sure we will need to track `docutils` version in fixtures at this stage.